src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -1436,7 +1502,7 @@ export class BrightScriptDebugSession extends BaseDebugSession {
                 }
 
                 //is at debugger prompt
-            } else {
+            } else if (args.expression.trim()) {
```

Comment:
Maybe add a comment about why we fall into this if block. (prevent showing hover info when hovering over whitespace-only sections of code)